nanocloud/errors/errors.go

```diff
@@ -61,16 +61,39 @@ var (
 		"The request is not valid.",
 	}
 
-	WindowsNotOnline = &apiError{
+	InvalidMarchineStatus = &apiError{
 		0x000007,
-		http.StatusServiceUnavailable,
-		"Windows is not available.",
+		http.StatusBadRequest,
+		"The specified machine status is not valid.",
 	}
 
-	NeedFirstConnection = &apiError{
+	UnableToUpdateMachineStatus = &apiError{
 		0x000008,
-		http.StatusServiceUnavailable,
-		"The features requires a first connection to Windows to be available",
+		http.StatusInternalServerError,
+		"The machine status cannot be updated.",
 	}
 
+	UnableToCreateTheMachine = &apiError{
+		0x000009,
+		http.StatusInternalServerError,
+		"The machine cannot be created.",
+	}
+
+	UnableToTerminateTheMachine = &apiError{
+		0x00000A,
+		http.StatusInternalServerError,
+		"The machine cannot be terminated.",
+	}
+
+	UnableToRetrieveMachineList = &apiError{
+		0x000010,
+		http.StatusInternalServerError,
+		"The machine list cannot be retrieved.",
+	}
+
+	WindowsNotOnline = &apiError{
+		0x000011,
```

Comment:
This error code is expected not to change since frontend is checking for it sometimes.